src/compass/utils/h5_helpers.py

```diff
@@ -567,13 +567,27 @@ def metadata_to_h5group(parent_group, burst, cfg):
         add_dataset_and_attrs(burst_meta_group, meta_item)
 
     # Add parameters group in processing information
+    tropo_corr = True if cfg.weather_model_file is not None else False
+    tec_corr = True if cfg.tec_file is not None else False
     par_meta_items = [
-        Meta('ellipsoidalFlatteningApplied', cfg.geocoding_params.flatten,
-             "If True, CSLC-S1 phase has been flatten with respect to a zero height ellipsoid",
-             {'units':'unitless'}),
-        Meta('topographicFlatteningApplied', cfg.geocoding_params.flatten,
-             "If True, CSLC-S1 phase has been flatten with respect to topographic height using a DEM",
-             {'units': 'unitless'}),
+        Meta('ellipsoidal_flattening_applied', bool(cfg.geocoding_params.flatten),
+             "If True, CSLC-S1 phase has been flatten with respect to a zero height ellipsoid"),
+        Meta('topographic_flattening_applied', bool(cfg.geocoding_params.flatten),
+             "If True, CSLC-S1 phase has been flatten with respect to topographic height using a DEM"),
+        Meta('bistatic_delay_applied', bool(cfg.lut_params.enabled),
+             "If True, bistatic delay correction has been applied"),
+        Meta('azimuth_fm_rate_applied', bool(cfg.lut_params.enabled),
+             "If True, azimuth FM-rate mismatch correction has been applied"),
+        Meta('geometry_doppler_applied', bool(cfg.lut_params.enabled),
+             "If True, geometry steering doppler correction has been applied"),
+        Meta('solid_earth_tides_applied', bool(cfg.lut_params.enabled),
+             "If True, solid Earth tides correction has been applied"),
+        Meta('static_troposphere_applied', bool(cfg.lut_params.enabled),
+             "If True, troposphere correction based on a static model has been applied"),
+        Meta('ionosphere_tec_applied', tec_corr,
+             "If True, ionosphere correction based on TEC data has been applied"),
+        Meta('troposphere_weather_model_applied', tropo_corr,
+             "If True, troposphere correction based on weather model has been applied")
```

Comment:
```suggestion
        Meta('dry_troposphere_weather_model_applied', tropo_corr,
             "If True, dry troposphere correction based on weather model has been applied")
        Meta('wet_troposphere_weather_model_applied', tropo_corr,
             "If True, wet troposphere correction based on weather model has been applied")
```
To be consistent #128 which separates both dry and wet.